node/resolvers/Mutations/Organizations.ts

```diff
@@ -137,13 +144,32 @@ const Organizations = {
       created: now,
     }
 
+    if (settings?.autoApprove) {
+      organizationRequest.status = 'approved'
+    }
+
     try {
       const result = await masterdata.createDocument({
         dataEntity: ORGANIZATION_REQUEST_DATA_ENTITY,
         fields: organizationRequest,
         schema: ORGANIZATION_REQUEST_SCHEMA_VERSION,
       })
 
+      if (settings?.autoApprove) {
+        Organizations.updateOrganization(
```

Comment:
This won't work. At this point you've only created an Organization Request. An Organization does not yet exist, so it cannot be updated. 

My suggestion would be to move the logic for creating an approved organization out from the `updateOrganizationRequest` resolver and into its own function that can be called from both that resolver and this one. 

If `settings.autoApprove` === true, this resolver should create the organization request with status = 'approved' instead of 'pending' (line 142 above) and then trigger the function to create an organization / create the default cost center / grant access to the organization admin user / send the appropriate emails.